packages/platform-core/src/errors/amplify_error.ts

```diff
@@ -100,17 +100,34 @@ export abstract class AmplifyError<T extends string = string> extends Error {
 
   static fromError = (
     error: unknown
-  ): AmplifyError<'UnknownFault' | 'CredentialsError'> => {
+  ): AmplifyError<
+    'UnknownFault' | 'CredentialsError' | 'InvalidCommandInputError'
+  > => {
     const errorMessage =
       error instanceof Error
         ? `${error.name}: ${error.message}`
         : 'An unknown error happened. Check downstream error';
 
     if (error instanceof Error && isCredentialsError(error)) {
-      return new AmplifyUserError('CredentialsError', {
-        message: errorMessage,
-        resolution: '',
-      });
+      return new AmplifyUserError(
+        'CredentialsError',
+        {
+          message: errorMessage,
+          resolution:
+            'Ensure your AWS credentials are correctly set and if required refreshed.',
```

Comment:
> if required refreshed

Does it mean "If required, refresh"? or "If `required` refreshed"? or "if requires a refresh"?

Reply:
NVM. I saw it here https://github.com/aws-amplify/amplify-backend/pull/1739#discussion_r1676323199

Reply:
Updated.